libs/components/lookup/src/lib/modules/lookup/lookup.component.spec.ts

```diff
@@ -3378,6 +3406,235 @@ describe('Lookup component', function () {
         await expectAsync(document.body).toBeAccessible(axeConfig);
       });
     });
+
+    // for testing non-async search args being passed around correctly
+    describe('search args (non-async)', () => {
+      // to test the passing of the 'context' arg
+      describe('context', () => {
+        beforeEach(() => {
+          // setting it to be multiselect is probably not required to test this, but it is the most common use case for using the optional 'context' arg
+          component.setMultiSelect();
+        });
+        describe('search function filters', () => {
+          let searchFilterFunctionSpy: jasmine.Spy;
+
+          it('should return popover context', fakeAsync(() => {
+            // creating the spy function that is pretending to be a search filter without any real functionality, because none is needed to simulate the initial search using a searchFilter
+            searchFilterFunctionSpy = jasmine.createSpy('searchFilterPopover');
+
+            component.customSearchFilters = [searchFilterFunctionSpy];
+            // peforming a search in the popover view should provide the search filter function with a 'popover' context
+            fixture.detectChanges();
+            tick();
+            performSearch('s', fixture);
+            expect(searchFilterFunctionSpy).toHaveBeenCalledWith(
+              's',
+              jasmine.objectContaining({
+                name: jasmine.any(String),
+              }),
+              jasmine.objectContaining({
+                context: 'popover',
+              })
+            );
+          }));
+
+          describe('show more modal', () => {
+            let modalService: SkyModalService;
+
+            beforeEach(fakeAsync(() => {
+              modalService = TestBed.inject(SkyModalService);
+
+              fixture.detectChanges();
+              tick();
+            }));
+
+            // This is necessary as due to modals being launched outside of the test bed they will not
+            // automatically be disposed between tests.
+            afterEach(fakeAsync(() => {
+              // NOTE: This is important as it ensures that the modal host component is fully disposed of
+              // between tests. This is important as the modal host might need a different set of component
+              // injectors than the previous test.
+              modalService.dispose();
```

Comment:
I made a recent change to modals so that manually disposing of them in unit tests is no longer required.

Reply:
fixed alongside in: [abstract Spy creation to helper functions](https://github.com/blackbaud/skyux/pull/381/commits/4dd35175b7154079e5b977b2e548ed1257120dda)